### DISCUSSION TITLE
preface/1-1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#868] [Announcements]
# preface/1-1

简介 简介 星空皮肤站 StarSkin，致力于为所有Minecraft玩家提供皮肤和披风，支持Github和Microsoft登录。星空皮肤站支持authlib-injector，使用皮肤站账号登录启动器。欢迎大家的加入，同时希望更多的玩家为皮肤站上传更多资源。 官网 ： https://star-skin.cn 有关地址 文档 ：docs.star...

https://docs.star-skin.cn/preface/1-1.html

**Comment by SRY-CTB** [comment]
测试
